proposals/boost/boost-maven/boost-boms/boost-mp21-apis-bom/pom.xml

```diff
@@ -0,0 +1,119 @@
+        <metrics-version>1.1</metrics-version>
+        <jwt-version>1.1</jwt-version>
+        <openapi-version>1.0</openapi-version>
+        <rest-client-version>1.1</rest-client-version>
+        <opentracing-version>1.2</opentracing-version>
+
+    </properties>
+
+	<dependencyManagement>
+		<dependencies>
+	        <dependency>
+	            <groupId>javax.enterprise</groupId>
+	            <artifactId>cdi-api</artifactId>
+	            <version>${cdi-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.ws.rs</groupId>
+	            <artifactId>javax.ws.rs-api</artifactId>
+	            <version>${jaxrs-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.json</groupId>
+	            <artifactId>javax.json-api</artifactId>
+	            <version>${jsonp-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.annotation</groupId>
+	            <artifactId>javax.annotation-api</artifactId>
+	            <version>${annotation-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.config</groupId>
+	            <artifactId>microprofile-config-api</artifactId>
+	            <version>${config-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.fault-tolerance</groupId>
+	            <artifactId>microprofile-fault-tolerance-api</artifactId>
+	            <version>${ft-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.health</groupId>
+	            <artifactId>microprofile-health-api</artifactId>
+	            <version>${health-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.metrics</groupId>
+	            <artifactId>microprofile-metrics-api</artifactId>
+	            <version>${metrics-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.jwt</groupId>
+	            <artifactId>microprofile-jwt-auth-api</artifactId>
+	            <version>${jwt-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.openapi</groupId>
+	            <artifactId>microprofile-openapi-api</artifactId>
+	            <version>${openapi-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.rest.client</groupId>
+	            <artifactId>microprofile-rest-client-api</artifactId>
+	            <version>${rest-client-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.eclipse.microprofile.opentracing</groupId>
+	            <artifactId>microprofile-opentracing-api</artifactId>
+	            <version>${opentracing-version}</version>
+	        </dependency>
+	        <dependency>
```

Comment:
Should the boosters be included in the API BOM?

Reply:
i don't think so - the booster dependencies are listed in the individual framework boms, which also import the api-boms 

ie: booster-ee8-bom lists all the individual booster dependencies for ee8 (at the correct versions) and imports the booster-ee8-apis-bom - i think that is for satisfying and separating both compile only targets and the boost package runtime target.

Reply:
I'm a bit confused. I think Matt is saying that the boosters should not be included in the API boms, whereas in the line that is referenced they are. I agree that they should be removed.

Reply:
Oh, ok, I see what you are saying - i am not sure how that happened, might have been a cut and paste issue. I will remove them.